code/modules/power/tesla/coil.dm

```diff
@@ -130,9 +130,10 @@
 		tesla_zap(src, 5, power_produced, tesla_flags, shocked_things)
 		var/datum/bank_account/D = SSeconomy.get_budget_account(ACCOUNT_ENG_ID)
 		if(D)
-			D.adjust_money(min(power_produced, 3))
+			D.adjust_money(min(power_produced, 3)*10)
 		if(istype(linked_techweb))
-			linked_techweb.add_point_type(TECHWEB_POINT_TYPE_DEFAULT, min(power_produced, 3)) // x4 coils with a pulse per second or so = ~720/m point bonus for R&D
+			linked_techweb.add_point_type(TECHWEB_POINT_TYPE_DEFAULT, min(power_produced, 3)*10)
+			linked_techweb.add_point_type(TECHWEB_POINT_TYPE_DISCOVERY, min(power_produced, 3)*10) // x4 coils with a pulse per second or so = ~7200/m point bonus for R&D
```

Comment:
7200 points per minute? Thats insane (For money too, 7200 a minute is just ridiculous)

Reply:
Alright, I'll turn those down. By how much though?

Reply:
It should be like 400

Reply:
Too little, it was 720 before and that was slow as hell.

Reply:
There, I adjusted the values and uploaded new videos with them